gcm_toolkit/tests/test_manipulations.py

```diff
@@ -164,3 +164,27 @@ def test_horizontal_overturning(all_nc_testdata):
     assert hasattr(dsi, "psi")
     assert (psi == dsi.psi).all()
     assert set(dsi.psi.dims) == {"Z", "time", "lat", "lon"}
+
+
+def test_extend_upward(all_nc_testdata):
+    """Create a minimal gcm_toolkit object and test both methods of extending
+    the upper atmospheric temperature."""
+
+    dirname, expected = all_nc_testdata
+
+    tools = GCMT()
+    tools.read_reduced(data_path=dirname)
+    dsi = tools.get_models()
+
+    T_iso = tools.extend_upward(1e-8, method='isothermal')
+
+    assert len(T_iso.Z) == len(dsi.Z) + 20
+    assert min(T_iso.Z) == 1e-8
+
+    T_the = tools.extend_upward(1e-8, method='thermosphere',
+                                  T_therm=10000, p_therm_high=1e-6)
+
+    assert len(T_the.Z) == len(dsi.Z) + 20
+    assert min(T_the.Z) == 1e-8
+    assert (T_the.sel(Z=slice(1e-6,1e-8)) < 10000).all()
+    assert (T_the.sel(Z=slice(1e-6,1e-8)) >= T_iso.sel(Z=slice(1e-6,1e-8))).all()
```

Comment:
That would be a nice improvement but I'm also ok with this version.

My bigger problem is that, according to codecov, your testing hits virtually no line in gcm_toolkit/utils/interface.py. Could you add testing to the PACInterface class as well?

Reply:
Thanks for looking over this. 

Yeah, I realise that the PAC interface is not tested, but this was a conscious decision. Most of the routines do similar stuff, namely writing out files. I spent a bit of time figuring out how to test for such functionality, but it involves setting up some mock I/O stuff that I frankly don't think is worth the time and effort. Especially since I'll be one of the only people using it. 

In short: I don't want to write tests for it, and I hope that's okay :) Let me know what you think.

Reply:
No, I agree! That's perfectly fine @robinbaeyens! I was only referring to the tests of extend_upwards 😊